pkgs/by-name/re/readexpro/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "readexpro";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "ThomasJockin";
+    repo = "readexpro";
+    #url = "https://github.com/ThomasJockin/readexpro.git";
+    rev = "563dfbb36ae45e52ec50829b016ce724ac2fca70";
+    #sparseCheckout = [
+    #  "/fonts"
+    #];
```

Delete the commented out lines and maybe add an explanation to why we're not using a tagged release (in this case it's because there isn't one), for people who might look at this in the future 
```suggestion
	# Upstream repository does not have any tagged releases
    rev = "563dfbb36ae45e52ec50829b016ce724ac2fca70";
```